backend/src/routes/auth.js

```diff
@@ -60,7 +54,7 @@ router.post('/refresh', [
   return res.status(200).json(refresh);
 });
 
-router.use('/token', auth.refreshJWT, auth.updateDBFromToken, async (req, res) => {
+router.use('/token', auth.refreshJWT, auth.updateDBFromToken, auth.getUserAcronyms, (req, res) => {
```

Comment:
Don't see that auth.getUserAcronyms does anything of benefit.

Reply:
auth.getUserAcronyms main purpose is to insert an acronyms attribute holding an array of acronyms a user has access to into the user object response of /auth/token. This is needed for the front-end to still be able to load its state and know what acronyms a user has access to.